### ISSUE TITLE
Unable to move while MOAR tablet is equipped and active.

### ISSUE THREAD

**Issue by doublestop** [klangzwang/MoarFactory#33]
When I use the MOAR tablet, I find that I am unable to move my character. I wonder if it has to do with key bindings, as I am an ESDF mover. It appears as though the E key is hard-coded in some way with respect to the mod, rather than the mod respecting the keybindings I have set. For example, my interaction bind is set to the A key rather than the default E key (which I have set to move forward) but the mod doesn't appear to respect it.

Steps to reproduce should be:

1. Rebind move forward to E.
2. Return to game and activate the MOAR tablet.
3. Try to move forward using the E key. It will not work.

Otherwise, this mod is fantastic!

**Comment by rcamarda390** [comment]
Yup, there have been other posts about this..i seem to recall a file can be edited to override this, but I am not sure.

**Comment by doublestop** [comment]
Thank you for the heads up! If you happen to recall how to override, please let me know. I'd love to give this mod another shot!
